go-controller/pkg/node/node.go

```diff
@@ -197,19 +198,9 @@ func setupOVNNode(node *kapi.Node) error {
 		if err != nil {
 			return err
 		}
-		uuid, _, err := util.RunOVNSbctl("--data=bare", "--no-heading", "--columns=_uuid", "find", "Encap",
-			fmt.Sprintf("chassis_name=%s", systemID))
-		if err != nil {
-			return err
-		}
-		if len(uuid) == 0 {
-			return fmt.Errorf("unable to find encap uuid to set geneve port for chassis %s", systemID)
-		}
-		_, stderr, errSet := util.RunOVNSbctl("set", "encap", uuid,
-			fmt.Sprintf("options:dst_port=%d", config.Default.EncapPort),
-		)
-		if errSet != nil {
-			return fmt.Errorf("error setting OVS encap-port: %v\n  %q", errSet, stderr)
+
+		if err = libovsdbops.MutateEncapOptions(sbClient, systemID, map[string]string{"dst_port": fmt.Sprintf("%d", config.Default.EncapPort)}); err != nil {
```

Comment:
Can `dst_port` option already exist with a different value that we would still want to change? If that's the case a mutate won't do it. Mentioning it just in case.

Reply:
Hrm it wont? Are you saying we only check the existence of the key before mutating? NOT the keys and values in the map? (I need to go check)